examples/pom.xml

```diff
@@ -51,6 +51,11 @@
       <artifactId>protobuf-java-util</artifactId>
       <version>${protobuf.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.9</version> <!-- prevent downgrade via protobuf-java-util -->
```

Comment:
Btw, how did you noticed this?

Reply:
CI failed with a RequireUpperBoundDeps failure.